clustered/client/src/main/java/org/ehcache/clustered/client/internal/store/EventualServerStoreProxy.java

```diff
@@ -104,6 +106,13 @@ public boolean removeInvalidationListener(InvalidationListener listener) {
   }
 
   @Override
+  public void close() {
+    this.entity.removeResponseListener(EhcacheEntityResponse.ServerInvalidateHash.class);
```

Comment:
See my other comment on the method declaration - this is wrong as soon as you have more than one clustered cache in a cache manager.